api/migrations/20200709183253-loadMemDocs.js

```diff
@@ -331,12 +332,12 @@ async function createMineDocument(nrpti, nrptiMine, collection, collectionDoc, n
   document.write = [utils.ApplicationRoles.ADMIN, utils.ApplicationRoles.ADMIN_BCMI];
 
   // upload to s3
-  await s3.upload({
-    Bucket: OBJ_STORE_BUCKET,
-    Key: s3Key,
-    Body: minioObject.Body,
-    ACL: 'authenticated-read'
-  }).promise()
+  // await s3.upload({
+  //   Bucket: OBJ_STORE_BUCKET,
+  //   Key: s3Key,
+  //   Body: minioObject.Body,
+  //   ACL: 'authenticated-read'
+  // }).promise()
```

Comment:
I don't think any of these changes should be part of this PR - it's an old migration.

Reply:
Good catch thanks